new-lamassu-admin/src/components/SearchFilter.js

```diff
@@ -1,38 +1,56 @@
 import { makeStyles } from '@material-ui/core'
+import * as R from 'ramda'
 import React from 'react'
 
 import Chip from 'src/components/Chip'
-import { P } from 'src/components/typography'
+import { ActionButton } from 'src/components/buttons'
+import { P, Label2 } from 'src/components/typography'
 import { ReactComponent as CloseIcon } from 'src/styling/icons/action/close/zodiac.svg'
+import { ReactComponent as FilterIcon } from 'src/styling/icons/button/filter/white.svg'
+import { ReactComponent as ReverseFilterIcon } from 'src/styling/icons/button/filter/zodiac.svg'
 
 import { chipStyles, styles } from './SearchFilter.styles'
 
 const useChipStyles = makeStyles(chipStyles)
 const useStyles = makeStyles(styles)
 
-const SearchFilter = ({ filters, onFilterDelete, setFilters }) => {
+const capitalize = R.converge(R.concat(), [
+  R.compose(R.toUpper, R.head),
+  R.tail
+])
```

Comment:
Like this approach, gj!

Reply:
I thought it was clean too, but I just realized we already have a fun to do this in the codebase. 🥲